.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,9 @@
+- [ ] I've added a new note in the `CHANGELOG.md` file, according to docs: https://iver-wharf.github.io/#/development/writing-changelog
```

Comment:
I'm in the midst of writing this page. You're welcome to leave comment on the format of this todo item as well as the URL path, even though the page does not exist yet.